lib/data-builder.js

```diff
@@ -16,9 +16,11 @@ module.exports = function buildResponseData(err, isDebugMode) {
   fillStatusCode(data, err);
 
   if (typeof err !== 'object') {
-    data.statusCode = 500;
-    data.message = '' + err;
-    err = {};
+    err = {
+      name: 'Error',
```

Comment:
Can we omit `name` in this case? Will it cause any failures in existing tests, either here or in strong-remoting?